docs/source/user_guide/peptides_proteins.rst

```diff
@@ -13,7 +13,6 @@ some commonly used operations like chemical formula, weight, and isotope distrib
 The example below shows how amino acid sequences can be created and how basic mass calculations are conducted.
 
 .. code-block:: python
-    :linenos:
```

Comment:
Why are these removed?

Reply:
Instead of removing this line, I fixed in next commits of format issue.